### ISSUE TITLE
Enums in DynamicEPackage

### ISSUE THREAD

**Issue by ewoudwerkman** [pyecore/pyecore#56]
I'm trying to assign an enum literal to a variable as follows:

```
... #load meta model, etc.
esdl = DynamicEPackage(esdl_model)

windturbine = esdl.WindTurbine(type=esdl.WindTurbineEnumType.WIND_ON_LAND)
```
This doesn't work, it says that _AttributeError: 'EEnum' object has no attribute 'WIND_ON_LAND'_
If I do the following it works:
```
windturbine = esdl.WindTurbine(type=esdl.WindTurbineEnumType.from_string('WIND_ON_LAND'))
```
But this is for developers more error prone.
In the static version, where classes are first generated, assigning it directly using the (name of the) literal works. Is this by intention? Reading the code of  ```EEnum.__init___``` it reads:
```
self.__setattr__(lit_name, literal)
```
So that should work, right?

**Comment by aranega** [comment]
Hi @ewoudwerkman ,

Indeed, as you saw, it works, but only if you init the `EEnumeration` passing the literals to the constructor. In the current state, what happens is that during the XMI deserialization, the `eLiterals` collection is directly handled. It then does not go throught the constructor. I patched the code and it works now. I will push it quickly to develop with new tests!

Thanks a lot for the bug report!

**Comment by ewoudwerkman** [comment]
Thanks for the fix!

**Comment by aranega** [comment]
Hi @ewoudwerkman ! Sorry for the delay, everything was fixed on the `develop` branch. I released a new version `0.10.1` that includes the fix ;)
Thanks for you patience!

**Comment by ewoudwerkman** [comment]
It works as expected!
